pkg/minikube/cluster/cluster.go

```diff
@@ -437,7 +440,13 @@ func GetVMHostIP(host *host.Host) (net.IP, error) {
 		}
 		return ip, nil
 	case "virtualbox":
-		ip, err := getIPForInterface("vboxnet0")
+		out, err := exec.Command("vboxmanage", "showvminfo", "minikube", "--machinereadable").Output()
+		if err != nil {
+			return []byte{}, errors.Wrap(err, "Error running vboxmanage command")
+		}
+		re := regexp.MustCompile(`hostonlyadapter2="(.*?)"`)
```

Comment:
If we end up adding more of these regex matches we should probably just add a few unit tests for them.  This is probably fine for now.